### ISSUE TITLE
Multi-Stage builds incorrectly expect a version label

### ISSUE THREAD

**Issue by Hades32** [replicatedhq/dockerfilelint#183]
E.g.
```dockerfile
FROM alpine:3.13 as base

RUN apk add --no-cache nodejs~=14 npm~=14

FROM base as builder

COPY package.json package-lock.json ./
# ...
```
Complains about `base` not having a label
